tutorial/20_recipes/010_reevaluate_best_trial.py

```diff
@@ -0,0 +1,93 @@
+"""
+.. _reevaluate_best_trial:
+
+Re-evaluate the best trial
+==========================
+
+You might re-evaluate the objective with the best hyperparameters again
+after the hyperparameter optimization.
+For example,
+
+- You investigate deeply the behavior of ML model with the best hyperparameter.
+- You train the model on the whole dataset that combine the training dataset and the validation dataset.
```

Comment:
I think that this is not a common use case. How about replacing it with the following?

You have optimized with Optuna using a partial dataset because the training time is long. After the hyperparameter tuning, you train the model using the whole dataset.

Reply:
Thank you for your feedback! The use-case is mentioned in [the deep learning book](https://www.deeplearningbook.org/), but I agree with you. It is not the common way. I'll replace the second example with your suggestion.